src/foam/u2/FloatView.js

```diff
@@ -82,7 +82,13 @@ foam.CLASS({
       view.sub(function() {
         if ( preventFeedback ) return;
         preventFeedback = true;
-        data.set(self.textToData(view.get()));
+        // check bounds on data update and set to boundary values if out of bounds
+        let val = view.get();
+        if ( self.min !== null && val < self.min )
```

Comment:
foam.Number has a `clamp` function in `stdlib.js` Not sure if it would work in this case but if it does then we could remove these checks

Reply:
Oh never mind. min or max can be undefined in this case. This is fine

Reply:
good to know that exists, thanks